channels/drive/client/statvfs.c

```diff
@@ -36,7 +38,7 @@ int statvfs(const char *path, struct statvfs *buf)
 	DWORD lpTotalNumberOfClusters;
 
 	len = MultiByteToWideChar(CP_ACP, 0, path, -1, NULL, 0);
-	unicodestr = (LPWSTR) malloc(len);
+	unicodestr = (LPWSTR) malloc(len*sizeof(TCHAR));
```

Comment:
I think this should be sizeof(WCHAR) rather than sizeof(TCHAR) - otherwise we still have a buffer overrun when we compile with TCHAR=CHAR